meal_mate/app/src/main/java/com/android/engineer/mealmate/view/utils/constants/MealConstants.kt

```diff
@@ -0,0 +1,14 @@
+package com.android.engineer.mealmate.view.utils.constants
+
+// Temporary constants - Once API integrated these static images will be removed.
```

Comment:
still, this is a data stuff, can be kept in data layer, no worries, just a heads up

Reply:
Yes, moved to data layer..